### PR TITLE
Remove colon after all labels & add required class

### PR DESCRIPTION
**Pull request by FlyersPh9**
- Removed colon after all input labels.
- Added `.iui-required` to `.iui-label` that appends a red * after.
- Moved some `<h2>` outside of the test `<div>`s & `<section>`s.
- Removed the image from header tests so I could get a pass.

Screen Shot 2021-05-12 at 12 53 48 PM
Screen Shot 2021-05-12 at 12 53 55 PM

### CODE REVIEW

**Comment by FlyersPh9** [comment]
Unsure why the header is showing a change within the user icon's picture, there should be no change there.  This same issue has happened before.
Screen Shot 2021-05-12 at 12 52 26 PM

**Comment by mayank99** [comment]
> Screen Shot 2021-05-12 at 12 52 26 PM

I've also seen that happen when working with header 😕 Might have to hide that image

File: backstop/tests/header.html [hunk 1]
```diff
@@ -577,7 +574,6 @@ <h2>Slim</h2>
             <user-icon
               type="2"
               size="small"
-              showPlaceholder="true"
```
Comment (mayank99) [inline backstop/tests/header.html:580]: I don't know about this change. I would have first tried to re-run tests to see if they'd pass

File: src/inputs/labeled-inputs.scss [hunk 2]
```diff
@@ -41,8 +41,13 @@
       color: t(iui-text-color);
     }
 
-    &::after {
-      content: ':';
+    &.iui-required {
```
Comment (veekeys) [inline src/inputs/labeled-inputs.scss:44]: I would prefer us handling required state (same as disabled, readonly) versus class..

Comment (mayank99) [reply src/inputs/labeled-inputs.scss:44]: Two problems:
- `:required` is set on `input` which comes after `.iui-label`, and there is no easy CSS selector for that.
- We are not using `input` everywhere, so separate class makes sense. Think of LabeledSelect (which is not using native `select`).

File: backstop/tests/header.html (hunk 1 above)
Comment (FlyersPh9) [reply backstop/tests/header.html:580]: I was trying to hide the image in the scenario JS, I tried:
`hideSelectors: ['user-icon']`
`hideSelectors: ['.iui-user-icon img']`
`hideSelectors: ['img']`
But wasn't able to hide the image from the tests. Am I doing it wrong?

File: src/inputs/labeled-inputs.scss (hunk 2 above)
Comment (veekeys) [reply src/inputs/labeled-inputs.scss:44]: hmm...
well maybe, then will handle required state in react, I guess..
Now the only concern is that it is a breaking visual change in react too...

**Review by veekeys — APPROVED** [review]

**Pull request merged**